maps/us-healthcare-system-capacity/js/main.js

```diff
@@ -166,31 +167,36 @@ var indicators = [
     propertyInData: "Staffed All Beds",
     label: "Staffed All Beds",
     colors: ["#e0ecf4", "#8856a7"],
-    displayAsPercent: false
+    displayAsPercent: false,
+    radii: [[1, 20], [5, 50]]
```

Comment:
This is the radii that will be used for the circle styling.
The first element is the min and max values for the first zoom level stop, etc.